daemon/api.go

```diff
@@ -2627,13 +2627,17 @@ func postApps(c *Command, r *http.Request, user *auth.UserState) Response {
 		return InternalError("no services found")
 	}
 
-	chg, err := servicestate.Change(st, appInfos, &inst)
+	ts, err := servicestate.Change(st, appInfos, &inst)
```

Comment:
Can we please rename this as servicestateControl?  Probably less confusing now that we're not returning a Change anymore.

Reply:
Done (in the prerequisite branch #4065).